### PR TITLE
update useScaffoldEventSubsciber example

### PR DESCRIPTION
**Pull request by technophile-04**
### Description : 

Updated the example of `useScaffoldEventSubscriber` with a new one, we forgot to update it 

Also added a note at https://github.com/scaffold-eth/se2-docs/commit/6040216cf712134494dae4c6ab632c6442ebc8e0 for #18 

Please feel free to suggest better explanation 😅🙌

### CODE REVIEW

**Review by Pabl0cks — APPROVED** [review]
Great explanation Shiv! 🙌 The link to react docs and internal link to `pollingInterval` are just brilliant! gj

I just did a small tweak because the horizontal scroll was bugging me a lot, and fixed a couple of typos.

Not sure if "destructed" was a typo, if it is correct just change it again ♥. Wish I had native english :)

File: docs/hooks/useScaffoldEventSubscriber.md [hunk 1]
```diff
@@ -11,11 +11,27 @@ useScaffoldEventSubscriber({
...
+        newGreeting
+      );
+    });
   },
 });
 ```
 
-This example subscribes to the `GreetingChange` event emitted by the `YourContract` smart contract, and logs the parameters emitted by the event to the console whenever it is emitted. The `listener` function accepts the parameters emitted by the event, and can be customized according to your needs.
\ No newline at end of file
+This example subscribes to the `GreetingChange` event emitted by the `YourContract` smart contract, and logs the parameters emitted by the event to the console whenever it is emitted. The `listener` function accepts an array of `logs` that occurred during the [ `pollingInterval` ](/deploying/deploy-nextjs-app#--pollinginterval) and each array items contains `args` property which can be destructured to get the parameters emitted by the event, this function can customized according to your needs.
```
Comment (edakturk14) [inline docs/hooks/useScaffoldEventSubscriber.md:31]:
```suggestion
This example subscribes to the `GreetingChange` event emitted by the `YourContract` smart contract and logs the parameters from the event to the console when it's emitted. The `listener` function accepts an array of `logs` that occurred during the [ `pollingInterval` ](/deploying/deploy-nextjs-app#--pollinginterval) and each array items contains an `args` property which can be destructured to get the parameters emitted by the event, this function can customized according to your needs.
```

Comment (edakturk14) [reply docs/hooks/useScaffoldEventSubscriber.md:31]: small change for readability. please feel free to update as you see fit!

**Review by edakturk14 — COMMENTED** [review]
Thank you so much @technophile-04!! Looks great to me, made small comment

**Comment by technophile-04** [comment]
Tysm all for the fixes and suggestions love it !!! Merging this 🙌

**Pull request merged**